### ISSUE TITLE
Support both syntaxes for input fields

### ISSUE THREAD

**Issue by hugo187** [laposa/testflow#137]
**Description**

Can we allow the old syntax for input fields as well? Sometimes description isn't necessary (e.g. field name `email` is descriptive enough).

````yaml
 input:
      <name>: Some string
````

Keep supporting the new syntax as well.

````yaml
  input:
      <name>:
        description: what is this input field about
        value: some-example-value-that-could-be-copied-into-clipboard
````

**Comment by hugo187** [comment]
@Szkandy @Kuchtin I am still getting the same error. Could you please double check it?

<img width="1630" alt="image" src="https://github.com/user-attachments/assets/2b79a7bb-9be4-406a-ab8e-245ab9ba96a4" />

**Comment by MartinMiksovskyLaposa** [comment]
Hey guys,
unfortunately I'm getting this after clicking on "Complete manual tests".

![image](https://github.com/user-attachments/assets/bf7e7009-6de5-469b-b7ff-9a9a6e638ed4)

**Comment by Szkandy** [comment]
fixed by #147
